fbw-a32nx/src/systems/fmgc/src/flightplanning/A380AircraftConfig.ts

```diff
@@ -144,22 +144,22 @@ const engineModelParams: EngineModelParameters = {
   table1506: [
     [0, 0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9],
     [0, 0.015, -0.15, 0, 0, 0, 0, 0, 0, 0, 0],
-    [20, 0.1286, -0.008, 0.015, 0.0337152, -0.0207408, -0.0447408, -0.0684924, -0.2470092, -0.3784788, -0.5864604],
-    [25, 0.155, 0.0236, 0.018, 0.0364848, -0.0115116, -0.0318852, -0.0503988, -0.1815936, -0.3194448, -0.5268336],
-    [30.0, 0.16, 0.16, 0.02, 0.039, 0.01, -0.01, -0.02, -0.107, -0.195, -0.428],
-    [35.0, 0.315, 0.151, 0.093, 0.055, 0.03, 0.013, -0.002, -0.083, -0.164, -0.394],
-    [40.0, 0.396, 0.195, 0.149, 0.07, 0.043, 0.05, 0.03, -0.051, -0.121, -0.342],
-    [45.0, 0.472, 0.3, 0.188, 0.11, 0.07, 0.1, 0.06, -0.012, -0.069, -0.275],
-    [50.0, 0.543, 0.373, 0.254, 0.13, 0.125, 0.16, 0.11, 0.033, -0.01, -0.196],
-    [55.0, 0.611, 0.448, 0.324, 0.18, 0.155, 0.21, 0.21, 0.085, 0.055, -0.107],
-    [60.0, 0.714, 0.528, 0.402, 0.25, 0.165, 0.22, 0.26, 0.15, 0.124, -0.012],
-    [65.0, 0.792, 0.615, 0.489, 0.32, 0.21, 0.23, 0.28, 0.27, 0.196, 0.087],
-    [70.0, 0.88, 0.712, 0.586, 0.38, 0.28, 0.235, 0.31, 0.34, 0.267, 0.184],
-    [75.0, 0.982, 0.821, 0.695, 0.43, 0.39, 0.3, 0.32, 0.36, 0.39, 0.27],
-    [80.0, 1.092, 0.941, 0.811, 0.55, 0.45, 0.4, 0.4, 0.38, 0.44, 0.37],
-    [85.0, 1.23, 1.07, 0.932, 0.66, 0.57, 0.48, 0.55, 0.48, 0.49, 0.45],
-    [90.0, 1.388, 1.206, 1.049, 0.75, 0.68, 0.56, 0.64, 0.62, 0.57, 0.59],
-    [95.0, 1.538, 1.34, 1.153, 0.875, 0.75, 0.6, 0.68, 0.69, 0.66, 0.69],
+    [20, 0.1286, -0.008, 0.015, 0.0008, -0.023, -0.061, -0.081, -0.135, -0.21, -0.3],
```

Comment:
Typo? It is `0.0008` in this table, but `0.008` in engines.cfg. I don't know which one is correct.

```suggestion
    [20, 0.1286, -0.008, 0.015, 0.008, -0.023, -0.061, -0.081, -0.135, -0.21, -0.3],
```

Reply:
engines.cfg should be correct since it was a copy paste of the engines.cfg file I actually used.